vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/screenshot/NativeHeaderAwareCoordsProvider.java

```diff
@@ -19,25 +19,39 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.vividus.selenium.mobileapp.MobileAppWebDriverManager;
+import org.vividus.selenium.screenshot.AbstractAdjustingCoordsProvider;
+import org.vividus.ui.context.IUiContext;
 
 import ru.yandex.qatools.ashot.coordinates.Coords;
-import ru.yandex.qatools.ashot.coordinates.WebDriverCoordsProvider;
 
-public class NativeHeaderAwareCoordsProvider extends WebDriverCoordsProvider
+public class NativeHeaderAwareCoordsProvider extends AbstractAdjustingCoordsProvider
 {
     private static final long serialVersionUID = 2966521618709606533L;
 
     private final transient MobileAppWebDriverManager mobileAppWebDriverManager;
+    private final transient IUiContext uiContext;
 
-    public NativeHeaderAwareCoordsProvider(MobileAppWebDriverManager mobileAppWebDriverManager)
+    public NativeHeaderAwareCoordsProvider(MobileAppWebDriverManager mobileAppWebDriverManager, IUiContext uiContext)
     {
+        super(uiContext);
         this.mobileAppWebDriverManager = mobileAppWebDriverManager;
+        this.uiContext = uiContext;
```

Comment:
it's better to introduce `protected` getter in `AbstractAdjustingCoordsProvider `